cpp/data/sql_updates/vufind.7

```diff
@@ -0,0 +1 @@
+ALTER TABLE vufind.user ADD journal_subscription_format ENUM ('meistertask');
```

Comment:
- Since the whole subscription logic is in the IxTheo module, we should move this field to the ixtheo_user table.
- If we use an ENUM it would be better to have the values 'default' and 'meistertask', and add DEFAULT 'default' for the migration of existing rows.
- You should also provide a 2nd PR in the tuefind repo to add DB changes to the installer.
- To be compatible with all MySQL & MariaDB versions, you should use "ADD COLUMN" instead of "ADD".

Reply:
As discussed.